app/controllers/workshops_controller.rb

```diff
@@ -11,13 +11,18 @@ def render_workshop
     begin
       workshop = params[:workshop]
       path = workshops_path.join(workshop, 'README.md')
+      url = request.url
 
       @title = workshop.humanize.titleize
 
       @metadata_page_name = @title
       @metadata_page_category = 'Workshop'
```

Comment:
`@metadata_page_name` and `@metadata_page_category` both don't need to be set now that we're doing this tracking on the backend.

I'd actually remove those two variables entirely from the application (so from `helpers/application_helper.rb`).